### ISSUE TITLE
Error installing localhost dev server

### ISSUE THREAD

**Issue by IslandEdition** [jmcbroom/airtable-tracker-toolkit#2]
I am super excited to try out this toolkit, thank you for much for taking the time to release it, and to put together the guide.

I am running into an error when starting the development server.

Initially the error was PowerShell saying "The term 'run' is not recognized as the name of a cmdlet..." I was entering "run npm install" without quotes. I did a little reading, someone suggested not using the word run, so I ran "npm install" without quotes and it started running the install, but the failed partway through. 

The install looks like it started to fall apart right around where this was returned in Terminal:

The node binary used for scripts is C:\Program Files (x86)\Nodist\bin\node.exe but npm is using C:\Program Files (x86)\Nodist\v-x64\11.13.0\node.exe itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

I am wondering if anyone has had trouble getting their dev server running. I am not really sure where to start with troubleshooting. Thanks.

**Comment by IslandEdition** [comment]
To update, the error was occurring right before the installer was suppose to download some files from Github - Sharp. That was resolved by uninstalling Node.js and reinstalling with version 12.12.0 from this website: https://nodejs.org/es/blog/release/v12.13.0/

Now the installer stops, it doesn't return an error, but it looks as if option dependencies were skipped. There are quite a few listed, but the few say I must install them myself.

←[0m←[37;40mnpm←[0m ←[0m←[30;43mWARN←[0m ←[0m←[35moptional←[0m SKIPPING OPTIONAL DEPENDENCY: @netlify/routing-local-proxy-linux-x64@^0.34.1 (node_modules\netlify-cli\node_modules\@netlify\routing-local-proxy\node_modules\@netlify\routing-local-proxy-linux-x64):

←[0m←[37;40mnpm←[0m ←[0m←[30;43mWARN←[0m ←[0m←[35mnotsup←[0m SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for @netlify/routing-local-proxy-linux-x64@0.34.1: wanted {"os":"linux","arch":"x64"} (current: {"os":"win32","arch":"x64"})

←[0m←[37;40mnpm←[0m ←[0m←[30;43mWARN←[0m←[35m←[0m @mdx-js/loader@2.1.1 requires a peer of webpack@>=4 but none is installed. You must install peer dependencies yourself.

←[0m←[37;40mnpm←[0m ←[0m←[30;43mWARN←[0m←[35m←[0m tsutils@3.21.0 requires a peer of typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta but none is installed. You must install peer dependencies yourself.

**Comment by IslandEdition** [comment]
I'll throw one more update in. I deleted the 'development-tracker' files on my computer, then followed the steps to download them again, and modify the .env file. This time, I downloaded the the files to my Windows C, rather than my backup D (which is an SSD).

This time the install ran for quite a while. It looked quite promising, but again it returned some errors. I'll just post the log file. If anyone can think of anything I can try, I'd appreciate it.

Running Windows 11 x64.

[2022-04-27T21_12_27_619Z-debug.log](https://github.com/jmcbroom/local-development-tracker-toolkit/files/8577887/2022-04-27T21_12_27_619Z-debug.log)

**Comment by IslandEdition** [comment]
A quick update to say that I have resolved my issues and have successfully started my localhost server.

The issue, I believe, was with Nodist. This morning I removed everything I installed on my PC yesterday, and started over again from Page 11 (Setting up the site) in [The Guide](https://local-development-tracker-toolkit.netlify.app/guide/setting-up-the-site). The only thing I did different this time was use NVM-Windows.

Looking forward to configuring my tracker, and setting it out into the wild. Thanks again for making your work public.
